steps/unable-to-enter/second-failed-attempt.tsx

```diff
@@ -0,0 +1,221 @@
+  LabelProps,
+  List,
+  PageAnnouncement,
+  Textarea,
+} from "lbh-frontend-react/components";
+import { useRouter } from "next/router";
+import React from "react";
+import {
+  ComponentDatabaseMap,
+  ComponentWrapper,
+  DynamicComponent,
+  makeDynamic,
+  StaticComponent,
+} from "remultiform/component-wrapper";
+import { TransactionMode } from "remultiform/database";
+import { Checkboxes, CheckboxesProps } from "../../components/Checkboxes";
+import { makeSubmit } from "../../components/makeSubmit";
+import failedAttemptReasonCheckboxes from "../../helpers/failedAttemptReasonCheckboxes";
+import getProcessRef from "../../helpers/getProcessRef";
+import keyFromSlug from "../../helpers/keyFromSlug";
+import useDataValue from "../../helpers/useDataValue";
+import PageSlugs from "../../steps/PageSlugs";
+import PageTitles from "../../steps/PageTitles";
+import ProcessDatabaseSchema from "../../storage/ProcessDatabaseSchema";
+import Storage from "../../storage/Storage";
+
+const storeSecondVisitDate = async (): Promise<void> => {
+  const db = await Storage.ProcessContext?.database;
+
+  if (!db) {
+    console.error("Unable to find process database");
+
+    return;
+  }
+
+  const processRef = keyFromSlug()();
+  const date = new Date().toISOString();
+
+  await db.transaction(
+    ["unableToEnter"],
+    async (stores) => {
+      const unableToEnter =
+        (await stores.unableToEnter.get(processRef)) ||
+        ({} as ProcessDatabaseSchema["schema"]["unableToEnter"]["value"]);
+
+      await stores.unableToEnter.put(processRef, {
+        ...unableToEnter,
+        secondFailedAttempt: {
+          ...unableToEnter.secondFailedAttempt,
+          date,
+        },
+      });
+    },
+    TransactionMode.ReadWrite
+  );
+};
+
+const PreviousFailedAttemptsAnnouncement: React.FunctionComponent = () => {
+  const router = useRouter();
+  const processRef = getProcessRef(router);
+  const firstFailedAttempt = useDataValue(
+    Storage.ProcessContext,
+    "unableToEnter",
+    processRef,
+    (values) =>
+      processRef ? values[processRef]?.firstFailedAttempt : undefined
+  );
+  const dateString =
+    firstFailedAttempt.loading || !firstFailedAttempt.result?.date
+      ? "Loading..."
+      : formatDate(new Date(firstFailedAttempt.result?.date), "d MMMM yyyy");
+
+  const reasonValue =
+    firstFailedAttempt.loading || !firstFailedAttempt.result?.value
+      ? "Loading..."
+      : firstFailedAttempt.result?.value;
+
+  const reasonLabel = failedAttemptReasonCheckboxes
+    .filter((checkbox) => reasonValue.includes(checkbox.value))
+    .map((checkbox) => checkbox.label);
+
+  return (
+    <PageAnnouncement title={"Previous attempt"}>
+      <List
+        items={[
+          <React.Fragment key="first">
+            First failed attempt: {dateString}
+          </React.Fragment>,
+          <React.Fragment key="second">
+            Reason: <List items={reasonLabel.map((label) => label)} />
+          </React.Fragment>,
```

Comment:
This created multiple list items for what should probably be a single item. Try:

```suggestion
          <React.Fragment key="first-attempt">
            <div>First failed attempt: {dateString}</div>
            <div>Reason: <List items={reasonLabel.map((label) => label)} /></div>
          </React.Fragment>,
```

Where the `div`s might be better as `Paragraph`s.